pkg/counter/prefixes.go

```diff
@@ -45,6 +45,44 @@ func NewPrefixLengthCounter(maxUniquePrefixes6, maxUniquePrefixes4 int) *PrefixL
 	}
 }
 
+// This is a bit ugly, but there's not a great way to define an IPNet without
+// parsing strings, etc.
+func createIPNet(ones, bits int) *net.IPNet {
+	return &net.IPNet{
+		Mask: net.CIDRMask(ones, bits),
+	}
+}
+
+type prefixTracker interface {
```

Comment:
It absolutely isn't important but leaving this private means godoc won't show it and so it will be slightly annoying to see how to meet `prefixTracker`